### PR TITLE
chore(changelog): improve header

### PR DESCRIPTION
**Pull request by Serpentiel**
## About this change—what it does

added a comment to improve the header

## Why this way

we forget to keep the header from time to time

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,10 @@ nav_order: 1
 
 # Changelog
 
-## [5.0.0] - YYYY-MM-DD
+<!-- Always keep the following header in place: -->
+<!-- [MAJOR.MINOR.PATCH] - YYYY-MM-DD -->
 
-- Migrate `aiven_service_integration` to the Plugin Framework
-- Deprecating `project_user`, `account_team` and `account_team_member` resources
```
Comment (byashimov) [inline CHANGELOG.md:12]: First we need to revert these changes.

Comment (Serpentiel) [reply CHANGELOG.md:12]: changelog is effective only when something is already released

**Review by byashimov — APPROVED** [review]
lgtm

**Review by byashimov — CHANGES_REQUESTED** [review]
Please resolve conflicts. Otherwise lgtm

**Review by byashimov — APPROVED** [review]
lgtm
